src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -657,16 +654,32 @@ export class GraphStyles {
     };
 
     const toFixedPercent = (num: number): string => {
+      num = safeNum(num);
       return `${trimFixed(num.toFixed(1))}%`;
     };
 
     const toFixedDuration = (num: number): string => {
+      num = safeNum(num);
       if (num < 1000) {
         return `${num.toFixed(0)}ms`;
       }
       return `${trimFixed((num / 1000.0).toFixed(2))}s`;
     };
 
+    // This due to us never having figured out why a tiny fraction of what-we-expect-to-be-numbers
+    // are in fact strings.  We don't know if our conversion in GraphData.ts has a flaw, or whether
+    // something else happens post-conversion.
+    const safeNum = (num: any): number => {
+      if (Number.isFinite(num)) {
+        return num;
+      }
+      if (typeof num === 'string' || num instanceof String) {
+        console.log(`Expected number but received string: |${num}|`);
+      }
+      // this will return NaN is the string is 'NaN' or any other non-number
```

Comment:
will return NaN ...if...the string
```suggestion
      // this will return NaN if the string is 'NaN' or any other non-number
```